bin/local-serve.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+
+function cli {
+  CKAN_CONTAINER=$(sh bin/docker-compose.sh ps -q ckan)
+      if [ "${#}" -ne 0 ]; then
+        docker exec $CKAN_CONTAINER sh -c '. "${APP_DIR}"/bin/activate; cd $APP_DIR;'" $*"
+      else
+        docker exec $CKAN_CONTAINER sh -c '. "${APP_DIR}"/bin/activate && cd $APP_DIR && sh'
+      fi
+}
+
+# Process Docker Compose configuration. This is used to avoid multiple
+# docker-compose.yml files.
+# Remove lines containing '###'.
+sed -i -e "/###/d" docker-compose.yml
+# Uncomment lines containing '##'.
+sed -i -e "s/##//" docker-compose.yml
+
+export CKAN_VERSION="2.11"
```

Comment:
Why hard-code this?

Reply:
this is for my local testing, its not used in cicd and i did not have ahoy/amzee installed